processing/data_collection/gazette/spiders/rn_macaiba.py

```diff
@@ -0,0 +1,76 @@
+import re
+
+import dateparser
+import datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMacaiba(BaseGazetteSpider):
+    start_date: dt.date
+    end_date: dt.date
+
+    TERRITORY_ID = "2407104"
+    EXTRA_ISSUE_PATTERN = r"Edição Extraordinária"
+    name = "rn_macaiba"
+    allowed_domains = ["macaiba.rn.gov.br"]
+
+    def __init__(self, start_date=None, end_date=None):
+        self.start_date = dt.date.today().replace(day=1)
+        self.end_date = dt.date.today()
+
+        if start_date is not None:
+            parsed_data = dateparser.parse(start_date)
+            if parsed_data is not None:
+                self.start_date = parsed_data.date()
+
+        if end_date is not None:
```

Comment:
If not done yet, maybe it should be done in the base class as well

Reply:
Done.